frontend/app/api.js

```diff
@@ -110,19 +112,42 @@ export function fetchUsers() {
   return ftch(baseUrl+usersUrl, header())
 }
 
-export function fetchCv(id) {
-  return ftch(`${baseUrl}${usersUrl}/${id}${cvUrl}`, header())
+const CV_FIELDS = `
+  userId
+  sections {
+    title
+    items {
+      title
+      description
+      when
+      organization
+      city
+    }
+  }
+`
+
+export function fetchCv() {
+  const query = `{
+    cv { ${CV_FIELDS} }
+  }
+  `
+  const url = `${baseUrl}${graphqlUrl}?query=${query}`
+  return ftch(url, { ...credentials() })
+    .then(res => Promise.resolve(res.data.cv))
 }
 
 export function updateCv(id, cv) {
```

Comment:
Also out of scope, but in the future I think we should split this file into several. Like one file per feature (cv, login, events etc) perhaps

Reply:
👍 yes, @jberglinds and I discussed this yesterday. Will be done at some point